### ISSUE TITLE
images not rendering in tmux popup windows?

### ISSUE THREAD

**Issue by rexroof** [seebye/ueberzug#137]
tmux recently added support for popup windows using `tmux display-popup`.

I notice that images rendered with ueberzug show up fine in a normal tmux pane, but they are missing when I try to render them inside a tmux popup window.

**Comment by seebye** [comment]
Well.. the implementation is based on the tmux pane and the environment variable TMUX_PANE isn't set if a program is executed in a popup.  
So ueberzug has no clue in which tmux client the popup is displayed in.

**Comment by rexroof** [comment]
thanks for looking into this.      can I manually set TMUX_PANE?  I'm guessing it wouldn't be aligned properly if I did that.   Is there any reasonable workaround?

**Comment by seebye** [comment]
No, it would not work as the offset of the dialog is still ignored.
